test/callback.jl

```diff
@@ -10,8 +10,7 @@
 # test/callback.jl
 # Testing callbacks
 #############################################################################
-using JuMP, MathProgBase
-using Base.Test
+using MathProgBase
```

Comment:
Having `using JuMP` allows to run the file separately. Just replace `Base.Test` by `Compat.Test`. It's better not to mix this PR with other changes